File: backend-dotnet/Application/Queries/Posts/GetPostLikesQuery.cs
using Application.DTOs;

namespace Application.Queries.Posts
{
    public class GetPostLikesQuery : IQuery<PagedResult<SimpleUserDto>>
    {
        public Guid PostId { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }
    }
}

File: backend-java/src/main/java/com/jerrygram/application/queries/posts/GetPostLikesQuery.java
package com.jerrygram.application.queries.posts;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class GetPostLikesQuery {
    private UUID postId;
    private int page;
    private int pageSize;
}